PWGHF/HFL/Tasks/taskSingleMuonReaderAssoc.cxx

```diff
@@ -0,0 +1,193 @@
+      const auto pt(muon.pt());
+      const auto charge(muon.sign());
+      const auto chi2(muon.chi2MatchMCHMFT());
+
+      if ((eta >= etaMax) || (eta < etaMin)) {
+        continue;
+      }
+
+      if ((rAbs >= rAbsMax) || (rAbs < rAbsMin)) {
+        continue;
+      }
+      if (pDca >= pDcaMax) {
+        continue;
+      }
+
+      // histograms after acceptance cuts
+      if (muon.has_matchMCHTrack()) {
+        auto muonType3 = muon.template matchMCHTrack_as<TMuons>();
+        auto Dpt = muonType3.pt() - pt;
+
+        singleMuon(pt, dcaXY, Dpt, chi2);
+        registry.fill(HIST("hMuAfterCuts"), pt, eta, dcaXY, rAbs, charge, chi2, Dpt);
+      }
+    }
+  }
+  template <typename TCollisions, typename TMuons>
+  void runMuonSelMcAssoc(ReducedMuonsAssoc const& assocs, TCollisions const& collisions, TMuons const&)
+  {
+    for (const auto& collision : collisions) {
+      registry.fill(HIST("hVtxZ"), collision.posZ());
+    }
+    for (const auto& assoc : assocs) {
+      auto muon = assoc.template reducedmuon_as<TMuons>();
+      if (muon.trackType() != trkType) {
+        continue;
+      }
+
+      const auto eta(muon.eta()), pDca(muon.pDca());
+      const auto rAbs(muon.rAtAbsorberEnd());
+      const auto dcaXY(RecoDecay::sqrtSumOfSquares(muon.fwdDcaX(), muon.fwdDcaY()));
+      const auto pt(muon.pt());
+      const auto charge(muon.sign());
+      const auto chi2(muon.chi2MatchMCHMFT());
+
+      if ((eta >= etaMax) || (eta < etaMin)) {
+        continue;
+      }
+
+      if ((rAbs >= rAbsMax) || (rAbs < rAbsMin)) {
+        continue;
+      }
+      if (pDca >= pDcaMax) {
+        continue;
+      }
+
+      // histograms after acceptance cuts
+      if (muon.has_matchMCHTrack()) {
+        auto muonType3 = muon.template matchMCHTrack_as<TMuons>();
+        auto Dpt = muonType3.pt() - pt;
+
+        singleMuon(pt, dcaXY, Dpt, chi2);
+        registry.fill(HIST("hMuAfterCuts"), pt, eta, dcaXY, rAbs, charge, chi2, Dpt);
+        if (muon.mcMask() == 0) {
+          registry.fill(HIST("hMuAfterCutsTrue"), pt, eta, dcaXY, rAbs, charge, chi2, Dpt);
+        }
+        if (muon.mcMask() == 128) {
+          registry.fill(HIST("hMuAfterCutsFake"), pt, eta, dcaXY, rAbs, charge, chi2, Dpt);
+        }
+      }
+    }
+  }
+  void processMuon(ReducedMuonsAssoc const& assocs, MyCollisions const& collisions, MyMuons const& muons)
+  {
+    runMuonSelAssoc(assocs, collisions, muons);
+  }
+  void processMuonMc(ReducedMuonsAssoc const& assocs, MyCollisions const& collisions, MyMcMuons const& muons)
+  {
+    runMuonSelMcAssoc(assocs, collisions, muons);
+  }
+
+  PROCESS_SWITCH(HfTaskSingleMuonReader, processMuon, "run muon selection with real data", true);
+  PROCESS_SWITCH(HfTaskSingleMuonReader, processMuonMc, "run muon selection with MC data", false);
+};
+
+WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
+{
+  return WorkflowSpec{
+    adaptAnalysisTask<HfTaskSingleMuonReader>(cfgc),
+  };
```

Comment:
Make it a single line.

Reply:
Hi Vit, thank you for the comments. Now the format of the code is fixed as you requested.

Reply:
However, if I change these to a single line, the clang-format check will not pass. I will keep it as before.

Reply:
Because you have a comma at the end, which should not be there. Remove it and make is a single line, please.